app/src/test/java/org/openobservatory/ooniprobe/test/TestAsyncTaskTest.java

```diff
@@ -178,14 +181,14 @@ public void runTest_withError() {
         when(mockedSuite.getTestList(any())).thenReturn(new AbstractTest[]{test});
         doThrow(new RuntimeException("")).when(test).run(any(), any(), any(), any(), anyInt(), any());
 
-        TestAsyncTask task = new TestAsyncTask(a, suiteList, runService, false);
+        TestAsyncTask task = new TestAsyncTask(a, suiteList,false);
 
         // Act
         task.execute();
         idleTaskUntilFinished(task);
 
         // Assert
-        verify(runService).stopSelf();
+        // verify(runService).stopSelf();
```

Comment:
If commenting out this line is meant to cause the test to succeed rather than failing, then I think it would be much more cleaner to just use the `Ignore` attribute like you did for the previous test.